sys/src/9/port/sysfile.c

```diff
@@ -717,11 +717,10 @@ mountfix(Chan *c, uint8_t *op, int32_t n, int32_t maxn)
 }
 
 static int32_t
-read(int fd, void *p, int32_t n, int64_t off)
+read(int ispread, int fd, void *p, int32_t n, int64_t off)
```

Comment:
Why is this first for read, but last for write?

Reply:
Also, I'd keep the 'sequential' name: it implies `ispread` would be true.

Reply:
I just follow the original definition.